### PR TITLE
Add guides for new onboarding wizards

### PR DESCRIPTION
**Pull request by iker-barriocanal**
At the moment, there're no docs of how to add new guides to the onboarding wizard.

### CODE REVIEW

**Review by rhcarvalho — APPROVED** [review]
Oh so many steps required, thanks for noting them down!

File: src/docs/contributing/onboarding-wizard.mdx [hunk 1]
```diff
@@ -12,6 +12,17 @@ The content for the onboarding wizard is located in `src/wizard`, and follows sl
 - It is exported as plain HTML (encoded as JSON), thus styles need to exist within Sentry itself.
 - The exposed frontmatter is specific to the wizard.
 
+## Adding Guides
+
+To add a guide of a new platform, follow the next steps:
```
Comment (PeloWriter) [inline src/docs/contributing/onboarding-wizard.mdx:17]:
```suggestion
To add a guide to a platform:
```

File: src/docs/contributing/onboarding-wizard.mdx [hunk 2]
```diff
@@ -12,6 +12,17 @@ The content for the onboarding wizard is located in `src/wizard`, and follows sl
...
+
+1. Write the guide, follow the guidelines of the [Directory Structure](#directory-structure) and [Page Format](#page-format).
```
Comment (PeloWriter) [inline src/docs/contributing/onboarding-wizard.mdx:19]:
```suggestion
1. Write the guide, follow both the guidelines of the [Directory Structure](#directory-structure) and [Page Format](#page-format) as well as the examples that currently exist in the directory; the most up to date is typically the wizard for the browser. Note that you do not directly copy the Getting Started page, as the context of this content differs.
```

File: src/docs/contributing/onboarding-wizard.mdx [hunk 3]
```diff
@@ -12,6 +12,17 @@ The content for the onboarding wizard is located in `src/wizard`, and follows sl
...
+2. If required, add an icon to [`platformicons`](https://github.com/getsentry/platformicons). If added, you also need to add a [platform to icon mapping](https://github.com/getsentry/platformicons/blob/master/src/platformIcon.tsx), bump the package version and update its version in repositories that use it ([`sentry`](https://github.com/getsentry/sentry) and [`docs`](https://github.com/getsentry/sentry-docs)).
```
Comment (PeloWriter) [inline src/docs/contributing/onboarding-wizard.mdx:20]:
```suggestion
2. If required, add an icon to [`platformicons`](https://github.com/getsentry/platformicons). If added, you also need to add a [platform to icon mapping](https://github.com/getsentry/platformicons/blob/master/src/platformIcon.tsx), bump the package version, and update its version in repositories that use it ([`sentry`](https://github.com/getsentry/sentry) and [`docs`](https://github.com/getsentry/sentry-docs)).
```

Comment (PeloWriter) [reply src/docs/contributing/onboarding-wizard.mdx:20]: Is there a reason we are pointing to the GH links rather than the directories?

**Review by PeloWriter — APPROVED** [review]
Thank you so much for adding these guidelines, @iker-barriocanal - super helpful to capture all the steps. 

One q - in the other guidelines, we point users to the directories (assuming they've cloned the repo). Could we do that here to be consistent?

**Pull request merged**